pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -133,9 +139,75 @@ func (c *PayloadAcceptedController) sync(ctx context.Context, key string) error
 		return err
 	}
 
+	originalCondition := conditions.GetReleasePayloadCondition(originalReleasePayload, v1alpha1.ConditionPayloadAccepted)
+	acceptedCondition, ok := computeReleasePayloadAcceptedCondition(originalReleasePayload, originalCondition)
+	if !ok {
+		return nil
+	}
+
+	if !conditions.ConditionChanged(originalCondition, acceptedCondition) {
```

Comment:
you want the library-go SetoperatorCondition like logic.

Reply:
Done!